web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx

```diff
@@ -261,6 +275,133 @@ export const NodeRightClickMenu = (props: {
           />
         </CustomScrim>
       )}
+      {scrimToShow === 'new-entity' && (
+        <div>
+          <CustomScrim
+            closeScrim={() => setScrimToShow('')}
+            header={`Create new entity`}
+            width={'30vw'}
+          >
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column',
+                justifyContent: 'space-between',
+              }}
+            >
+              <div style={{ display: 'block' }}>
+                <Label label={'Blueprint'} />
+                <BlueprintPicker
+                  onChange={(selectedType: string) =>
+                    setFormData({ type: selectedType })
+                  }
+                  formData={formData?.type || ''}
+                />
+              </div>
+              {loading ? (
+                <Button
+                  style={{
+                    marginTop: '40px',
+                    width: '200px',
+                    alignSelf: 'center',
+                  }}
+                >
+                  <Progress.Dots />
+                </Button>
+              ) : (
+                <Button
+                  disabled={formData?.type === undefined}
+                  style={{
```

Comment:
looks like this style is repeated quite a few times.

Maybe its possible to used a styled component?
(I'm not sure if this is supported with the EDS Button though...)

Reply:
Could not use styled-components, but used a config object :)